quranHome/src/androidMain/Kotlin/com/islamversity/quran_home/view/JuzView.kt

```diff
@@ -0,0 +1,44 @@
+package com.islamversity.quran_home.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import com.airbnb.epoxy.AfterPropsSet
+import com.airbnb.epoxy.CallbackProp
+import com.airbnb.epoxy.ModelProp
+import com.airbnb.epoxy.ModelView
+import com.islamversity.quran_home.databinding.NumberViewBinding
+import com.islamversity.quran_home.model.JozUIModel
+
+@ModelView(
+    autoLayout = ModelView.Size.MATCH_WIDTH_WRAP_HEIGHT,
+    saveViewState = true
+)
+class JuzView @JvmOverloads constructor(
+    context: Context,
+    attributeSet: AttributeSet? = null
+) : LinearLayout(context, attributeSet) {
+
+    private val binding = NumberViewBinding.inflate(LayoutInflater.from(context), this, true)
+
+    @ModelProp
+    lateinit var juz: JozUIModel
+
+    @AfterPropsSet
```

Comment:
TBH, We don't need this annotation.
we can create a function with `@ModelProp` and when passing juz we can set the values at the same time